src/constraint/encoding.ts

```diff
@@ -239,6 +239,35 @@ export const ENCODING_CONSTRAINTS: EncodingConstraintModel[] = [
       return true;
     }
   },{
+    name: 'scalePropertiesSupportedByChannel',
+    description: 'Not all scale properties are supported by all encoding channels',
+    properties: [Property.CHANNEL, Property.SCALE],
```

Comment:
properties here should include scale properties too, otherwise they won't get triggered.